Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:alpine
+WORKDIR /Rest-api
+COPY requirements.txt makefile student.py /Rest-api/ 
+RUN apk update
```

Comment:
Related to above comment, read about combining multiple `RUN` commands to speed up docker build process.

Reply:
I've combined multiple run commands in dockerfile.